docs/8_creating-provider-plugins.md

```diff
@@ -1,57 +1,54 @@
+---
+title: Building Serverless Provider Integrations
+description: todo
+layout: Page
+---
+
 # Building provider integrations
 
 Integrating different infrastructure providers happens through the standard plugin system.
 Take a look at the ["building plugins"](./building-plugins.md) documentation to understand how the plugin system works.
 
 ## Provider specific plugins
 
-You can add the providers name inside the constructor of your plugin. This makes it possible to only execute your
-plugins logic when the Serverless service uses the provider you've specified in your plugin.
+You can add the providers name inside the constructor of your plugin. This makes it possible to only execute your plugins logic when the Serverless service uses the provider you've specified in your plugin.
 
 ## Deployment
 
-Infrastructure provider plugins should bind to specific lifecycle events of the `deploy` command to compile the function
-and their events to provider specific resources.
+Infrastructure provider plugins should bind to specific lifecycle events of the `deploy` command to compile the function and their events to provider specific resources.
 
-### Deployment lifecycles
+### Deployment lifecycle
 
 Let's take a look at the [core `deploy` plugin](/lib/plugins/deploy) and the different lifecycle hooks it provides.
 
 The following lifecycle events are run in order once the user types `serverless deploy` and hits enter:
 
-- `deploy:cleanup`
```

Comment:
This lifecycle was added recently and should be here.

Maybe we should revisit the lifecycles before we merge this so that we have the most recent version in place.